tests/TestCase/ORM/EntityTest.php

```diff
@@ -713,6 +713,20 @@ public function testJsonSerialize()
     }
 
     /**
+     * Tests serializing an entity as PHP
+     *
+     * @return void
+     */
+    public function testPhpSerialize()
+    {
+        $data = ['name' => 'James', 'age' => 20, 'phones' => ['123', '457']];
+        $entity = new Entity($data);
+        $copy = unserialize(serialize($entity));
+        $this->assertInstanceOf(Entity::class, $copy);
+        $this->assertEquals($data, $copy->toArray());
```

Comment:
Is this adding coverage in general? It doesn't look related to the defect that is being fixed.